pkg/virt-controller/watch/clone/clone_test.go

```diff
@@ -721,6 +721,64 @@ var _ = Describe("Clone", func() {
 
 		})
 
+		Context("VirtualMachineTemplate Labels and annotations", func() {
+
+			type mapType string
+			const labels mapType = "labels"
+			const annotations mapType = "annotations"
+
+			expectTemplateLabelsOrAnnotations := func(m map[string]string, templateLabelOrAnnotation mapType) {
+				expectedVM := sourceVM.DeepCopy()
+
+				if templateLabelOrAnnotation == labels {
+					expectedVM.Spec.Template.ObjectMeta.Labels = m
+				} else {
+					expectedVM.Spec.Template.ObjectMeta.Annotations = m
+				}
+
+				expectVMCreationFromPatches(expectedVM)
+			}
+
+			DescribeTable("VirtualMachineTemplate filters", func(templateLabelOrAnnotation mapType) {
```

Comment:
Can you control what is tested with a simple bool instead of with a custom type?